### PR TITLE
fixes broken images

### PR DESCRIPTION
**Pull request by BellaSpence**

### CODE REVIEW

File: _docs/build-artifacts.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ whose contents will be saved.
 
 After the build finishes, everything in these directories is saved and linked to the build.
 
-![](asset://img/docs/artifacts.png)
+![]({{site.assets}}/img/docs/artifacts.png)
```
Comment (plainspace) [inline _docs/build-artifacts.md]: @ndintenfass @bellkev this image should be updated

**Comment by plainspace** [comment]
Looks like a couple of .DS_Store files got added but other than that lgtm! @ndintenfass?

**Comment by travis** [comment]
lgtm

**Comment by ndintenfass** [comment]
lgtm too -- go ahead and merge (but let's get .DS_Store in our .gitignore)

**Pull request merged**